corehq/apps/app_manager/views/modules.py

```diff
@@ -194,6 +202,35 @@ def _report_to_config(report):
         'all_reports': [_report_to_config(r) for r in all_reports],
         'current_reports': [r.to_json() for r in module.report_configs],
         'warnings': warnings,
+        'multimedia': {
```

Comment:
Most of this is identical to https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/views/view_generic.py#L172-L186 - any objection to DRYing it up?